ui/packages/lib/src/components/header/Header.js

```diff
@@ -15,6 +15,7 @@ import { HeaderUserMenu } from "./HeaderUserMenu";
 import { slugify } from "../../utils";
 import { NavDrawer } from "../nav_drawer";
 import "./Header.scss";
+import {Link} from "@reach/router";
```

Comment:
super nit: couple of the same thing, to add spaces around curly braces

Reply:
fixed and also updated `husky` to automatically format files according prettier configuration on pre-commit hook